spyder_line_profiler/spyder/widgets.py

```diff
@@ -424,7 +580,7 @@ def __init__(self, parent=None):
         self.setColumnCount(len(self.header_list))
         self.setHeaderLabels(self.header_list)
         self.clear()
-        self.itemActivated.connect(self.item_activated)
+        self.itemClicked.connect(self.onItemClick)
```

Comment:
Maybe this function should be called `on_item_clicked`?
```suggestion
        self.itemClicked.connect(self.on_item_clicked)
```
Also, is there some specific reason why `itemClicked` is better than `itemActivated`?

Reply:
Nope, I simply copied this piece of coded from somewhere else, so no reason.